### PR TITLE
papis: update to 0.9.

### PR DESCRIPTION
**Pull request by xaltsc**

### CODE REVIEW

**Comment by Chocimier** [comment]
Please put new dependencies in same PR, one commit per template.

**Comment by xaltsc** [comment]
Done

File: srcpkgs/papis/template [hunk 1]
```diff
@@ -1,21 +1,25 @@
...
 short_desc="Command-line based document and bibliography manager"
 maintainer="xaltsc <xaltsc@protonmail.ch>"
 license="GPL-3.0-or-later"
 homepage="https://github.com/papis/papis"
 distfiles="https://github.com/papis/papis/archive/v${version}.tar.gz"
-checksum=6c5493d5f7062a960bc2b1351ddde001ed585e407aa06e7c07092c61040c696a
+checksum=c49e53f843a53ccaaeaa24efa777c444efcfd541098fefcfa198ab99888b15e5
 
 pre_build() {
+        patch -p 1 -i "${FILESDIR}/update-vers1.patch"
```
Comment (Chocimier) [inline srcpkgs/papis/template]: Patches put into `patches` directory are applied without need to modify template. They need to be in -p0 format, so please remove `a/` from header.

**Comment by xaltsc** [comment]
It should be okay now, unless I missed something else.

**Pull request merged**